### ISSUE TITLE
Unclear phrase in chapter 2 about sequential consistency

### ISSUE THREAD

**Issue by ernesto-jimenez** [mixu/distsysbook#34]
Chapter 2 contains the following phrase:

> The difference seems immaterial, but it is worth noting that sequential consistency does not compose.

However, it's not clear what you mean when you say "sequential consistency does not compose".

**Comment by mixu** [comment]
Yeah, it's left a bit unexplained - if I we're rewriting the book today, I'd bring this particular section on the kinds of "strong" consistency models to be more in line with what Peter Bailis wrote recently http://www.bailis.org/blog/linearizability-versus-serializability/ also http://en.wikipedia.org/wiki/Sequential_consistency

I didn't quite want to bring in serializability vs. linearizability topic in yet at that point since serializability is a transactional DB related concept and much of what I write doesn't involve that stuff, but I'll try to revisit this section next time I'm updating the book so that it is a bit more clear.

**Comment by murraycu** [comment]
It would also be nice if it said whether "Sequential Consistency" and "Serializability" are the same thing.
